src/App.js

```diff
@@ -55,12 +55,24 @@ function App() {
               <ExperimentsPage />
             </Dashboard>
           )} />
+        <PrivateRoute
+          exact
+          path="/logout"
+          user={user}
+          component={() => (
+            <Dashboard>
+              <Logout />
+            </Dashboard>
+          )} />
         <Route exact path="/login">
           {user ? <Redirect to="/dashboard" /> : <Login />}
         </Route>
         <Route exact path="/signup">
           {user ? <Redirect to="/dashboard" /> : <Signup isDarkTheme={true} />}
         </Route>
+        <Route exact path="/logout">
+          {user ? <Redirect to="/logout" /> : <Redirect to="/login" />}
+        </Route>
```

Comment:
I think we can remove these 3 lines. By using `PrivateRoute` in lines 58-66, that component takes care of the redirecting when a user is not logged in.

Reply:
This bug was in the file `auth-provider`, I removed `setLoaded()` from reset values and it got resolved.